src/backend/services/chat.py

```diff
@@ -193,8 +196,10 @@ def is_custom_tool_call(chat_response: BaseChatRequest) -> bool:
     if chat_response.tools is None or len(chat_response.tools) == 0:
         return False
 
-    if chat_response.tools[0].description:
-        return True
+    # check if any of the tools is not in the available tools
+    for tool in chat_response.tools:
+        if tool.name not in AVAILABLE_TOOLS:
+            return True
```

Comment:
lol much better, thank you 😅

Reply:
lol this was broken because FE is now passing the whole tool object